diff --git a/lib/config/normalization.js b/lib/config/normalization.js
index a1acda77f74a70a90aee567af5bf7b7cf774eb3c..384952764a3820fadd30a11a520a7e3d5a5e3778 100644
--- a/lib/config/normalization.js
+++ b/lib/config/normalization.js
@@ -446,7 +446,8 @@ const getNormalizedWebpackOptions = config => {
 				hash: module.hash
 			})),
 			immutablePaths: optionalNestedArray(snapshot.immutablePaths, p => [...p]),
-			managedPaths: optionalNestedArray(snapshot.managedPaths, p => [...p])
+			managedPaths: optionalNestedArray(snapshot.managedPaths, p => [...p]),
+			unmanagedPaths: optionalNestedArray(snapshot.unmanagedPaths, p => [...p])
 		})),
 		stats: nestedConfig(config.stats, stats => {
 			if (stats === false) {
